pennylane/_device.py

```diff
@@ -411,9 +436,11 @@ def supports_operation(self, operation):
         if isinstance(operation, str):
 
             if operation.endswith(Operation.string_for_inverse):
-                return operation[
-                    : -len(Operation.string_for_inverse)
-                ] in self.operations and self.capabilities().get("inverse_operations", False)
+                in_ops = operation[: -len(Operation.string_for_inverse)] in self.operations
+                # TODO: update when all capabilities keys changed to "supports_inverse_operations"
+                supports_inv = (self.capabilities().get("supports_inverse_operations", False) or
+                                self.capabilities().get("inverse_operations", False))
+                return in_ops and supports_inv
```

Comment:
I know the change makes it a bit lengthy, but I wanted to use a structure where we just have to delete the ``or self.capabilities().get("inverse_operations", False)`` part once the keys are unified across external devices.